TreeTracker/UI/Views/AddTree/AddTreeViewController.swift

```diff
@@ -7,21 +7,37 @@
 //
 
 import UIKit
+import AVFoundation
 
 class AddTreeViewController: UIViewController, AlertPresenting {
 
+    @IBOutlet weak var previewView: UIView!
```

Comment:
[suggestion:](https://conventionalcomments.org/#labels) This can be private so it can't be accessed from outside the ViewController. Also [weak is no longer required](https://stackoverflow.com/a/34212435) on IBOutlet variables
```suggestion
    @IBOutlet private var previewView: UIView!
```